src/error.rs

```diff
@@ -226,6 +226,9 @@ pub enum BugId {
     ID004,
     ID005,
     ID006,
+    ID007,
+    ID008,
+    ID009,
```

Comment:
Why are you adding 3 IDs here but only 2 errors further down?

Reply:
This ID is a fallback for when the debug symbols are not available for the backtrace. They map to unique locations of the source, and we have 3 new instances:

https://github.com/FuelLabs/fuel-vm/blob/d559692b84e3bb6c1840a67806c71b2e92cea9d9/src/interpreter/blockchain.rs#L93-L99

https://github.com/FuelLabs/fuel-vm/blob/d559692b84e3bb6c1840a67806c71b2e92cea9d9/src/interpreter/blockchain.rs#L102-L105

https://github.com/FuelLabs/fuel-vm/blob/d559692b84e3bb6c1840a67806c71b2e92cea9d9/src/interpreter/blockchain.rs#L420-L422

Reply:
We are being over-cautious now, even with checks that logically can't panic. The idea is to reduce these checks when we enter the optimization phase